receiver/hostmetricsreceiver/internal/scraper/processscraper/ucal/cpu_utilization_calculator.go

```diff
@@ -57,12 +48,9 @@ func cpuUtilization(logicalCores int, startStats *cpu.TimesStat, startTime pcomm
 	systemUtilization := (endStats.System - startStats.System) / elapsedTime
 	ioWaitUtilization := (endStats.Iowait - startStats.Iowait) / elapsedTime
 
-	if normalizeProcessCPUUtilizationFeatureGate.IsEnabled() && logicalCores > 0 {
```

Comment:
Should we keep the `logicalCores > 0` guard to prevent `-/+ Inf` values? Having 0 `logicalCores` probably implies that something has gone wrong in determining the number of cores. Should we return an error instead?

Reply:
I was thinking about that while doing the code. I'll revert that part since it makes sense.